drawing_2.css

```diff
@@ -0,0 +1,56 @@
+.container{
+	width: 300px;
+	height: 100px;
+}
+.magenta{
+	background-color: #FF00FF;
+	width: 100px;
+	height: 100px;
+	float: right;
```

Comment:
Going back to this one again. If there is no serious reason to float stuff right, float it left for the sake of simplicity. Please change all styled `divs` to `float:left`